src/Components/Components/ref/Microsoft.AspNetCore.Components.netstandard2.0.cs

```diff
@@ -16,6 +16,33 @@ public abstract partial class AuthenticationStateProvider
         public abstract System.Threading.Tasks.Task<Microsoft.AspNetCore.Components.AuthenticationState> GetAuthenticationStateAsync();
         protected void NotifyAuthenticationStateChanged(System.Threading.Tasks.Task<Microsoft.AspNetCore.Components.AuthenticationState> task) { }
     }
+    public partial class AuthorizeView : Microsoft.AspNetCore.Components.AuthorizeViewCore
+    {
+        public AuthorizeView() { }
+        [Microsoft.AspNetCore.Components.ParameterAttribute]
+        public string Policy { [System.Runtime.CompilerServices.CompilerGeneratedAttribute]get { throw null; } }
+        [Microsoft.AspNetCore.Components.ParameterAttribute]
```

Comment:
Compared with the manual version of this code `Resource` is gone. @SteveSandersonMS - should that have changed?

Reply:
@rynowak looks like you're good here. That property is inherited from `AuthorizeViewCore`. See https://github.com/aspnet/AspNetCore/pull/12474/files#diff-c367512139edcf0bb6b1ce1b3b84ee4dR40

Reply:
Yeah, looks like there previously was a bug in the manual ref code, fixed by autogenerating it.